tests/core/selection/manual/optimizationlist.md

```diff
@@ -0,0 +1,14 @@
+@bender-tags: selection, 4.20.0, bug, 4931
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, elementspath, sourcearea, list, undo, selectall
+
+1. Select entire content via `Ctrl+A`.
+2. Delete content with backspace key.
+
+**Expected** The entire list content is removed from the editor.
+
+**Note** It is possible that list styling was not removed, so the first list item punctuation is in the content.
```

Comment:
I'd rephrase this note a little because the results of unfixed code are very similar – the first list item marker is preserved. The only difference is the fact that unfixed code produces additional whitespace before it.

So maybe:

```suggestion
**Note** It is possible that list styling was not removed, so the first list item punctuation is in the content **without any preceding whitespace**.
```